tests/HypothesisClient/Credentials/JWTSigningCredentialsTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace tests\eLife\HypothesisClient\Credentials;
+
+use eLife\HypothesisClient\Credentials\JWTSigningCredentials;
+use Firebase\JWT\JWT;
+use PHPUnit_Framework_TestCase;
+use Serializable;
+
+/**
+ * @covers \eLife\HypothesisClient\Credentials\JWTSigningCredentials
+ */
+class JWTSigningCredentialsTest extends PHPUnit_Framework_TestCase
+{
+    /**
+     * @test
+     */
+    public function it_can_be_serialized()
+    {
+        $credentials = new JWTSigningCredentials('foo', 'baz', 'authority', 300);
+        $this->assertInstanceOf(Serializable::class, $credentials);
+        $this->assertEquals($credentials, unserialize(serialize($credentials)));
+    }
+
+    /**
+     * @test
+     */
+    public function it_may_have_an_expire_time()
+    {
+        $credentials = new JWTSigningCredentials('foo', 'baz', 'authority');
+        $this->assertGreaterThan(0, $credentials->getExpireTime() - $credentials->getStartTime());
+        $credentials = new JWTSigningCredentials('foo', 'baz', 'authority', 100);
+        $this->assertEquals(100, $credentials->getExpireTime() - $credentials->getStartTime());
+    }
+
+    /**
+     * @test
+     */
+    public function it_may_have_a_start_time()
+    {
+        $credentials = new JWTSigningCredentials('foo', 'baz', 'authority');
+        $this->assertGreaterThanOrEqual(time(), $credentials->getStartTime());
+        $start_time = time();
```

Comment:
no need for this to be dynamic throughout tests, for better reproducibility it can be a constant like `1500000000`